test/Select-test.js

```diff
@@ -2129,6 +2129,66 @@ describe('Select', () => {
 
 	});
 
+	describe('with multi=true and onSelectResetsInput=false', () => {
+		it('should have retained inputValue after accepting selection', () => {
```

Comment:
it('should have retained inputValue after accepting selection if onSelectResetsInput=false ', () => {